Feature #148 fixing t1 flair lists

jan-petr opened the pull request:
* Bug in Lesion lists fixed.
* Merged the code for T1 and FLAIR lists and simplified the code.

Close #148

HenkMutsaerts reviewed (approved):
* >[ROI_T1_list; ROI_FLAIR_list]

Shouldn’t this be 

>[ROI_T1_list(:); ROI_FLAIR_list(:)]

To ensure a 1D vector?

* Would change iLesion to iROI (line 99 & 100) xASL_wrp_VisualQC_Structural.m)

jan-petr commented:
> > [ROI_T1_list(:); ROI_FLAIR_list(:)]
> 
> To ensure a 1D vector?

The adm_GetFileList output is a 1D vector, so not necessary. But I have changed it to avoid problems in the future.
 
> * Would change iLesion to iROI (line 99 & 100) xASL_wrp_VisualQC_Structural.m)

I have changed to iROI, and also to ROIFile, ROIIm on other places.

Please approve again (small things, but lets stick to the rules - with your addition to #151, I also discovered a bug on small additional commit).
